### PR TITLE
Import PEM 8.2 online help

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

This builds on #1771 by updating the PEM 8 online help to a conversion of the PEM 8.2 release.

@nidhibhammar please review these changes and merge them into your branch prior to merging that into develop. Should some changes here be undesirable, feel free to reject them (in whole or in part).

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [x] This PR changes existing content
- [x] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx [hunk 1]
```diff
@@ -104,6 +104,39 @@ After defining a new alert template, click the `Add/Change` button to save the d
...
+<figure><img src="../../images/alert_template_import_browse.png" class="align-center" alt="Alert Template - Import Browse" /><figcaption aria-hidden="true"><em>Alert Template - Import Browse</em></figcaption></figure>
+
+After selecting the file to import, you can select the following checkboxes:
+
+-   `skip exiting` - If you select this checkbox then it will skip the alert template if it already exists.
+-   `skip existing dependent probe` - The alert templates are dependent on probes. If you select this checkbox then it will skip the dependent probe if it already exists.
+
+If both the checkboxes are selected and the alert template already exists, then it skips importing the alert template with below message:
+
+<figure><img src="../../images/alert_template_skip_both.png" class="align-center" alt="Alert Template - Skip Message" /><figcaption aria-hidden="true"><em>Alert Template - Skip Message</em></figcaption></figure>
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx:130]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/01_pem_custom_probes.mdx [hunk 2]
```diff
@@ -13,7 +13,11 @@ Use the `Custom Probes` tab to create a new probe or modify an existing probe. A
...
 Use the `Show system probes?` switch to display the system probes on the `Custom Probes` tab.
 
-To modify an existing probe, click the `Edit` icon located to the left of a probe name. To create a new probe, click the `Add` icon in the upper-right corner of the tab.
+To modify an existing probe, click the `Edit` icon located to the left of a probe name.
+
+## Defining a New Probe
+
+To create a new probe, click the `Add` icon in the upper-right corner of the tab.
 
 <figure><img src="../../images/custom_probes_general.png" class="align-center" alt="Custom Probes - Add New Probe - General tab" /><figcaption aria-hidden="true"><em>Custom Probes - Add New Probe - General tab</em></figcaption></figure>
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/12_pem_manage_probes/01_pem_custom_probes.mdx:22]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/05_toc_pem_management_basics/10_managing_cluster_objects/06_role_reassign_dialog.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+title: "Role Reassign/Drop Own Dialog"
+---
+
+<div id="role_reassign_dialog" class="registered_link"></div>
+
+Use the *Reassign/Drop Own* dialog to change the ownership of database objects owned by a database role. This dialog instructs the system to change the ownership of database objects owned by any of the *old_roles* to *new_role*.
+
+The *Reassign/Drop Own* dialog organizes the Reassign & Drop role through General tab.
+
+<img src="../../images/role_reassign_dialog.png" class="align-center" alt="Reassign/Drop Own dialog" />
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/05_toc_pem_management_basics/10_managing_cluster_objects/06_role_reassign_dialog.mdx:11]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/02_query_tool.mdx [hunk 4]
```diff
@@ -173,19 +173,19 @@ If the server returns an error, the error message will be displayed on the `Mess
...
 ## Notifications Panel
 
-Use the `Notifications` tab to view the notifications using PostgreSQL `Listen/ Notify` feature. For more details see [PostgreSQL documentation \\&lt;https:// www.postgresql.org/docs/current/sql-listen.html>](<>).
+Use the `Notifications` tab to view the notifications using PostgreSQL `Listen/ Notify` feature. For more details see [PostgreSQL documentation](https:// www.postgresql.org/docs/current/sql-listen.html).
 
 Example:
 
 1.  Execute `LISTEN "foo"` in first `Query Tool` session
 
 <img src="../images/query_output_notifications_listen.png" class="align-center" alt="Query tool notifications listen" />
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/02_query_tool.mdx:182]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/04_editgrid/index.mdx [hunk 5]
```diff
@@ -80,6 +80,45 @@ Notes:
 -   `Data size:` For performance reasons, the viewer will render no more than 100000 geometries, totaling up to 20MB.
 -   `Internet access:` An internet connection is required for the Geometry Viewer to function correctly.
 
+**JSON Data Editor**
+
+A built in json editor is provided for *JSON/JSONB Data*.Double clicking on json/jsonb data type cell in data grid will open JSON Editor.Editor provides different mode to view and edit json data.
+
+<img src="../../images/json_editor_modes.png" class="align-center" alt="Json Editor" />
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/04_editgrid/index.mdx:87]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/07_psql_tool.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+<div id="psql_tool" class="registered_link"></div>
+
+PSQL tool allows user to connect to PostgreSQL/EDB Advanced server using psql terminal.
+
+-   Open PSQL Tool from Tools menu or PSQL tool button from browser tree or from context menu.
+-   PSQL will connect to the current connected database from browser tree.
+
+<img src="../images/psql_tool.png" class="align-center" alt="PSQL tool window" />
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/pem_online_help/08_toc_pem_developer_tools/07_psql_tool.mdx:12]: The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx (hunk 1 above)
Comment (josh-heyer) [reply product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx]: There were several hundred of these already in the PEM online help; I've rewritten all of them.

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**